playwright/tests/pageobjects/field/entity/FieldAssociation.ts

```diff
@@ -62,11 +61,11 @@ export class FieldAssociation {
     }
   }
 
-  async fillValues(cardinality: string, cascadeTypes: FieldAssociationCascadeTypes, mappedBy: string, removeOrphans: boolean) {
+  async fillValues(cardinality: string, cascadeTypes: FieldAssociationCascadeTypes, mappedBy: string | undefined, removeOrphans: boolean) {
     await this.collapsible.open();
     await this.cardinality.choose(cardinality);
     await this.fillCascadeTypes(cascadeTypes);
-    await this.mappedBy.locator.fill(mappedBy);
+    await this.mappedBy.choose(mappedBy);
```

Comment:
maybe better to check if undefined here than add this complexity to the select page object?

Reply:
Yes, makes much more sense.
The difference was between selecting the `emptyItem` (`choose('')`) and not making a change to the selection (`choose(undefined)`).